pages/enterprise/changelog.rst

```diff
@@ -2,6 +2,44 @@
 Changelog
 *********
 
+Graylog Enterprise 4.1.0
+========================
+
+Released: 2021-06-23
+
+Enterprise
+----------
+
+**Added**
```

Comment:
We can also mention https://github.com/Graylog2/graylog-plugin-enterprise/pull/2024 here.

Reply:
@linuspahl Thank you. Added!